app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,317 @@
+            android:layout_height="wrap_content"
+            android:layout_marginTop="12dp"
+            android:text="@string/genre"
+            android:textColor="@color/red"
+            android:textSize="14sp"
+            app:layout_constraintEnd_toEndOf="@id/right_guidline"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toBottomOf="@id/TV_movie_name" />
+
+        <ImageView
+            android:id="@+id/IV_rating_star_1"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="8dp"
+            android:src="@drawable/star_icon"
+            app:layout_constraintStart_toStartOf="@id/left_guidline"
+            app:layout_constraintTop_toBottomOf="@id/TV_movie_genre" />
+
+        <ImageView
+            android:id="@+id/IV_rating_star_2"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="3.25dp"
+            android:src="@drawable/star_icon"
+            app:layout_constraintStart_toEndOf="@id/IV_rating_star_1"
+            app:layout_constraintTop_toTopOf="@id/IV_rating_star_1" />
+
+        <ImageView
+            android:id="@+id/IV_rating_star_3"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="3.25dp"
+            android:src="@drawable/star_icon"
+            app:layout_constraintStart_toEndOf="@id/IV_rating_star_2"
+            app:layout_constraintTop_toTopOf="@id/IV_rating_star_1" />
+
+        <ImageView
+            android:id="@+id/IV_rating_star_4"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="3.25dp"
+            android:src="@drawable/star_icon"
+            app:layout_constraintStart_toEndOf="@id/IV_rating_star_3"
+            app:layout_constraintTop_toTopOf="@id/IV_rating_star_1" />
+
+        <ImageView
+            android:id="@+id/IV_rating_star_5"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="3.25dp"
+            android:src="@drawable/star_icon"
+            app:layout_constraintStart_toEndOf="@id/IV_rating_star_4"
+            app:layout_constraintTop_toTopOf="@id/IV_rating_star_1"
+            app:tint="@color/gray" />
+
+        <TextView
+            android:id="@+id/TV_review"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:layout_marginStart="12dp"
+            android:gravity="center"
+            android:text="@string/_125_reviews"
+            android:textColor="@color/gray"
+            android:textStyle="bold"
+            app:layout_constraintBottom_toBottomOf="@+id/IV_rating_star_1"
```

Comment:
maybe vice verse?
relate first star vertically depending on TV_review?
think about 1123121312312 reviews:(text for 2 or more lines)